### PR TITLE
Completed the assigned task and fixed one bug 

### PR DESCRIPTION
**Pull request by Aaisha-Rani**
* Added gymnasium Env id, File size on disk, and dataset group column to the Minari list local cli commands 

* Added dataset_group attribute, with  string Unknown as a placeholder value.

* Updated the Docs Data set standard and individual dataset pages.

* Fixed a bug  in point_maze_dataset.py

### CODE REVIEW

**Review by balisujohn — CHANGES_REQUESTED** [review]
When I run `minari list remote`I get a KeyError  for `dataset_group.` Since this is not implemented yet for our existing datasets, it should just default to Unknown if there is no `dataset_group`key present in metadata.

 `file_size` is not included in metadata, so it needs to be obtained a different way. I'm not 100% sure how, but this might be helpful https://stackoverflow.com/questions/50875461/google-cloud-storage-get-object-size-api.`file_size` also needs to be manually obtained for local datasets from the local hdf5 files, since there isn't an existing field where it's explicitly kept track of. 
 
To get the environment name, you should be able to access  `json.loads(dst_metadata["env_spec"])["id"]`

File: minari/utils.py [hunk 1]
```diff
@@ -323,6 +323,7 @@ def create_dataset_from_collector_env(
             "`author_email` is set to None. For longevity purposes it is highly recommended to provide an author email, or some other obvious contact information.",
             UserWarning,
         )
+
```
Comment (balisujohn) [inline minari/utils.py:326]: Looks like newlines added by accident.

File: docs/tutorials/dataset_creation/point_maze_dataset.py [hunk 2]
```diff
@@ -308,11 +308,12 @@ def __call__(self, env, obs, info, action=None, rew=None, terminated=None, trunc
         print(n_step)
         if dataset is None:
             dataset = minari.create_dataset_from_collector_env(collector_env=collector_env,
-                                                               dataset_name=dataset_name,
-                                                               algorithm_name="QIteration",
-                                                               code_permalink="https://github.com/Farama-Foundation/Minari/blob/main/docs/tutorials/dataset_creation/point_maze_dataset.py",
-                                                               author="Rodrigo Perez-Vicente",
-                                                               author_email="rperezvicente@farama.org")
+            dataset_id=dataset_name,
```
Comment (balisujohn) [inline docs/tutorials/dataset_creation/point_maze_dataset.py:311]: good catch

**Comment by younik** [comment]
Closing for inactivity; thanks a lot for doing this anyway!

**Pull request closed without merging**